Src/BootCamp.Chapter/Program.cs

```diff
@@ -6,7 +6,29 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            //Tom Jefferson is 19 years old, his weight is 50 kg and his height is 156.5 cm. 
+            string Name1 = "Tom Jefferson";
+            int Age1 = 19;
```

Comment:
You should read input from the console. Use ``Console.ReadLine()`` and ``int/float.Parse`` to convert console input to values that you can work with.